utils/src/deployer.rs

```diff
@@ -316,6 +318,22 @@ pub async fn deploy_light_client_proxy(
             .await?;
     }
 
+    // post deploy verification checks
+    assert_eq!(lc_proxy.getVersion().call().await?.majorVersion, 1);
+    assert_eq!(lc_proxy.owner().call().await?._0, admin);
+    assert_eq!(
+        lc_proxy.permissionedProver().call().await?._0,
+        prover.unwrap()
+    );
```

Comment:
should only check inside `if let Some(prover)` block

Reply:
good call, updated here https://github.com/EspressoSystems/espresso-network/commit/6fc2d192bd2dd2c8fa15f2fc926bf9c484988ea0